.github/workflows/main.yml

```diff
@@ -98,6 +100,164 @@ jobs:
 
       - name: Test Netbeans Build System
         run: ant -Dcluster.config=basic localtest
+
+  linux-php:
+    name: Linux build of PHP modules
+    runs-on: ubuntu-18.04
+    env:
+      OPTS: -quiet -Dcluster.config=php -Djavac.compilerargs=-nowarn -Dbuild.compiler.deprecation=false -Dtest-unit-sys-prop.ignore.random.failures=true
```

Comment:
@KacerCZ Remove the `-quiet` and `-Djavac.compilerargs=-nowarn -Dbuild.compiler.deprecation=false` params

Reply:
Removed.